apps/system/test/marionette/text_selection_test.js

```diff
@@ -121,6 +120,9 @@ marionette('Text selection >', function() {
       });
       test('click center input', function() {
         fakeTextselectionApp.longPress('DialogPositionCenterInput');
+        client.switchToFrame();
+        system.waitForKeyboard();
+        client.apps.switchToApp(FakeTextSelectionApp.ORIGIN);
```

Comment:
nit can you extract those 3 lines in a function (locally to this file), I'd suggest naming it `waitForKeyboard()`